book/C-git-commands/1-git-commands.asc

```diff
@@ -364,143 +364,146 @@ Pinakita natin paano gamitin ito para gumawa ng magandang changelog sa <<_distri
 
 ==== git imap-send
 
-The `git imap-send` command uploads a mailbox generated with `git format-patch` into an IMAP drafts folder.
+Ang `git imap-send` na utos ay nag-upload ng mailbox na nabuo sa pamamagitan ng `git format-patch` sa isang folder ng mga draft ng IMAP.
 
-We go through an example of contributing to a project by sending patches with the `git imap-send` tool in <<_distributed_git#_project_over_email>>.
+Tinalakay namin ang isang halimbawa ng pagbibigay ng kontribusyon sa isang proyekto sa pamamagitan ng pagpapadala ng mga patch gamit ang `git imap-send` na tool sa <<_distributed_git#_project_over_email>>.
 
 ==== git send-email
 
-The `git send-email` command is used to send patches that are generated with `git format-patch` over email.
+Ang `git send-email` na utos ay ginamit para magpadala ng mga patch kung saan ay nabuo ng `git format-patch` sa email.
 
 We go through an example of contributing to a project by sending patches with the `git send-email` tool in <<_distributed_git#_project_over_email>>.
+Tinalakay namin ang isang halimbawa ng pagkontribosyon sa isang proyekto sa pamamagitan ng mga patch gamit ang `git send-email` na tool sa <<_distributed_git#_project_over_email>>.
 
 ==== git request-pull
 
-The `git request-pull` command is simply used to generate an example message body to email to someone.
-If you have a branch on a public server and want to let someone know how to integrate those changes without sending the patches over email, you can run this command and send the output to the person you want to pull the changes in.
+Ang `git request-pull` na utos ay simpleng ginagamit upang bumuo ng isang halimbawa ng nilalaman ng mensahe upang mag-email sa isang tao.
+Kung meron kang isang branch na nasa pampublikong server at nais na ipaalam sa isang tao kung paano isama ang mga pagbabagong iyon na hindi na magpadala ng mga patch sa email, maaari mong patakbuhin ang utos na ito at ipadala ang output sa taong gusto mong kunin ang mga pagbabago.
 
-We demonstrate how to use `git request-pull` to generate a pull message in <<_distributed_git#_public_project>>.
+Pinakita namin kung paano gamitin ang `git request-pull` para makabuo ng pull na mensahe sa <<_distributed_git#_public_project>>.
 
 === External Systems
 
-Git comes with a few commands to integrate with other version control systems.
+Ang Git ay may ilang mga utos upang maisama ang iba pang mga sistema ng pagkontrol sa bersyon.
 
 ==== git svn
 
-The `git svn` command is used to communicate with the Subversion version control system as a client.
-This means you can use Git to checkout from and commit to a Subversion server.
+Ang `git svn` na utos ay ginagamit upang makipag-ugnayan sa sistema ng pagkontrol sa bersyon ng Subversion bilang isang kliyente.
+Ang ibig sabihin nito ay maaari mong gamitin ang Git upang mag-check out at mag-commit sa isang server ng Subversion.
```

Comment:
I think you should change `upang mag-check out at mag-commit sa isang server ng Subversion` to `upang mag-checkout mula at mag-commit patungo sa isang server ng Subversion`, though i could be wrong